packages/synapse-react-client/src/components/SynapseTable/SynapseTableRenderers.tsx

```diff
@@ -48,7 +48,7 @@ function RowSelectionCell(props: CellContext<Row, unknown>) {
       <Checkbox
         label={'Select row'}
         hideLabel={true}
-        checked={isRowSelected(row.original)}
+        checked={isRowSelected(row.original, rowSet!.headers)}
```

Comment:
`isRowSelected` now requires passing the selectColumns, since they aren't in stored in a  Jotai atom.